assets/src/edit-story/components/library/textLibrary.js

```diff
@@ -14,58 +14,105 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+
 /**
- * External dependencies
+ * Internal dependencies
  */
-import PropTypes from 'prop-types';
+import { DEFAULT_EDITOR_PAGE_HEIGHT } from '../../constants';
+import { editorToDataY } from '../../units/dimensions';
+import { Section, MainButton, Title, SearchInput, Header } from './common';
+import { FontPreview } from './text';
+
+const PRESETS = [
```

Comment:
Should these presents be defined here? Should there be defined in WP / PHP / REST API? cc @swissspidy ?

Reply:
_Eventually._ But one step at a time 🙂 

It is nice that this PR implements the bare minimum for this area and the needed UI components.

How exactly presets are going to work will be defined in a dedicated epic doc, and storage can then be discussed in #29.